### DISCUSSION TITLE
3DO Fz-10 (Anvil) adapter hasn't input

### DISCUSSION THREAD

**Discussion by CarloScalabrin** [darthcloud/BlueRetro#868] [General]
Hello everyone, I'm facing some problems to get Blureretro Core to work on the 3DO FZ-10 with Anvil chip. I am using pmgducati's PCB design. On Fz-1 and Fz-10 with DSP Clio it works normally, however on Fz-10 models with DSP Anvil, it does not work.  The core has power, the control pairs normally, but there is no button input. Any tips/suggestions?

**Comment by darthcloud** [comment]
I'm not sure what my FZ-10 revision is. I'll have to check.

@DoomyDoomer did you by any chance test with both version of the FZ-10?

**Reply by CarloScalabrin** [reply]
I did a test on fz-10 with "clio" and works perfectly.

**Comment by DoomyDoomer** [comment]
My FZ-10 appears to be a Japanese CLIO, 1994 model.

**Comment by CarloScalabrin** [comment]
Ok Guys, so do you think that it is necessary to open an issue? @darthcloud  and @DoomyDoomer

**Reply by DoomyDoomer** [reply]
Might be good to open a ticket just so it's tracked. I'm not sure what adapter (or DIY solution) you're using but FWIW, my adapter has been on the market for awhile now and I haven't heard of this issue before. It even works with the M2 prototypes.

**Comment by darthcloud** [comment]
I opened a bug to track this, but it will only get resolve if I can get my hand on the HW to troubleshoot.

**Comment by darthcloud** [comment]
Turn out this issue is related to the level shifter used.

AHC125 wont work

you needs 74AHCT1G125 or 74AHCT125

Note the AHCT you need all 4 letters.

I debugged one HB adapter and swapping the AHC125 for AHCT125 fixed the issue with the Anvil instently!!
